build.md

```diff
@@ -1 +1,5 @@
-# Building and running Piconot
\ No newline at end of file
+# Building and running Piconot
+
+1. Place your rules in a `.txt` file in the `piconot-external` folder.
+2. Update `piconot.scala` to point to the desires maze and rules.
+3. Compile and run the files.
```

Comment:
How? 
( I know how but you should be very specific. Assume you are trying to get someone who knows how to build things but is not in this class to build and run the program).